ui/tabs.js

```diff
@@ -410,27 +410,27 @@ return $.widget( "ui.tabs", {
 				if ( $( this ).closest( "li" ).is( ".ui-state-disabled" ) ) {
 					this.blur();
 				}
-			});
+			} );
 
 		this.tabs = this.tablist.find( "> li:has(a[href])" )
-			.attr({
+			.attr( {
 				role: "tab",
 				tabIndex: -1
-			});
+			} );
 		this._addClass( this.tabs, "ui-tab", "ui-state-default" );
 
-		this.anchors = this.tabs.map(function() {
-				return $( "a", this )[ 0 ];
-			})
-			.attr({
+		this.anchors = this.tabs.map( function() {
+			return $( "a", this )[ 0 ];
+		} )
```

Comment:
I think we either need to clarify this in our style guide or change how this is formatted. The current rule is:

> When a chain of method calls is too long to fit on one line, there must be one call per line, with the first call on a separate line from the object the methods are called on. If the method changes the context, an extra level of indentation must be used.

So, presumably this should be:

```js
this.anchors = this.tabs
    .map( function() {
        return $( "a", this )[ 0 ];
    } )
        .attr( {
            ...
        } );
```